.travis.yml

```diff
@@ -12,6 +12,8 @@ cache:
 
 before_script:
   - docker pull k8s.gcr.io/coredns:1.6.2
+  - docker pull registry:2
+  - go get -u github.com/google/go-containerregistry/cmd/crane
```

Comment:
Why the need to pull in crane here? Isn't this pulled into the tester docker file?

Reply:
Since we use crane to push a testing image to the custom registry in manager-script before running the tests, we need to get crane in here.

Reply:
Sidenote: If we go with pushing the image to registry with docker instead of crane, we have to add the registry as an insecure registry in daemon config.

Reply:
Fair point. I think we can use crane to push the image.
Happy with that approach of pre pushing an image. What is your view on self-contained tests so. Shouldn't the test init a push? To be fair we do prepare CoreDNS and the registry already.

Probably the usual way would be to mock more of the responses, but I like the real world tooling usage to run into real world issues at least in e2e tests. For unit tests mocking might be better.